config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
+  get 'films/index'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root to: 'pages#index'
+  root to: 'films#index'
+  resources :films, only: [:index]
```

Comment:
you just need `root to: 'films#index'` and (`get 'films/index'` or resources :films, only: [:index]) you can drop pages and one of routes that points to films index